File: src/spring/kotlin/hu/aut/meixner/dto/task/student/other/BlindMapTask.kt
package hu.aut.meixner.dto.task.student.other

import hu.aut.meixner.dto.task.common.MediaItemResponse
import hu.aut.meixner.dto.task.common.TaskTypeEnum
import hu.aut.meixner.dto.task.other.BlindMapTag
import hu.aut.meixner.dto.task.student.AssignTask

class BlindMapTask(
        override val taskId: Long,
        override val title: String,
        override val type: TaskTypeEnum = TaskTypeEnum.BlindMap,
        val image: MediaItemResponse,
        val tags: List<BlindMapTag>,
) : AssignTask()

File: src/angular/angular-meixner/src/swagger-api/model/blindMapTask.ts
import {BlindMapTag} from "./blindMapTag";
import {MediaItemResponse} from "./mediaItemResponse";
import {AssignTask} from "./assignTask";
import TypeEnum = AssignTask.TypeEnum;

export interface BlindMapTask {
  taskId: number;
  title: string;
  image: MediaItemResponse;
  tags: Array<BlindMapTag>;
  type: TypeEnum;
}
